concordium-node/src/consensus_ffi/helpers.rs

```diff
@@ -353,6 +357,9 @@ impl TryFrom<i64> for ConsensusFfiResponse {
             27 => Ok(ChainUpdateSequenceNumberTooOld),
             28 => Ok(ChainUpdateInvalidSignatures),
             29 => Ok(MaxBlockEnergyExceeded),
+            // 30 is used for`ResultImportStopped` which origins from a `SIGINT` or `SIGTERM`
+            // signal send to the node while catching up out of band.
+            31 => Ok(InsufficientFunds),
```

Comment:
Actually, looking at this now, the handling of StopImport is broken in the following ways.
- The error is not handled in the Rust side. This is generally fine since the node will stop at this point. But why return this status at all then? Why not just return "Ok".
- The return value of `importBlocks` is declared `u8` on the Rust side, but it is actually `u64` on the Haskell side.

I would recommend removing the "Import Stopped" message and returning Ok instead. And fixing the return value of `importBlocks` on the Rust side.

Reply:
I can do that in a followup PR if you prefer.

Reply:
I'll just fix it now I'm at it.